sdk/core/azure-core/src/main/java/com/azure/core/http/policy/HttpResponseLogger.java

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.http.policy;
+
+import com.azure.core.http.HttpResponse;
+import com.azure.core.util.logging.ClientLogger;
+import com.azure.core.util.logging.LogLevel;
+import reactor.core.publisher.Mono;
+
+import java.time.Duration;
+
+/**
+ * Manages logging HTTP responses in {@link HttpLoggingPolicy}.
+ */
+public interface HttpResponseLogger {
+    /**
+     * Retrieves the {@link LogLevel} used to log the current response.
+     *
+     * @param defaultLogLevel The default log level to use.
+     * @param response The HTTP response.
+     * @param responseDuration The duration between sending the request and receiving the response.
+     * @return The {@link LogLevel} used to log the current request.
+     */
+    LogLevel getLogLevel(LogLevel defaultLogLevel, HttpResponse response, Duration responseDuration);
```

Comment:
Same as above. Can we just have the `logResponse()` method in this interface?

Reply:
Same answer as above